pkg/config/env.go

```diff
@@ -90,4 +90,5 @@ var Config = struct {
 	JWTAuthSecret             string   `env:"FLAGR_JWT_AUTH_SECRET" envDefault:""`
 	JWTAuthNoTokenRedirectURL string   `env:"FLAGR_JWT_AUTH_NO_TOKEN_REDIRECT_URL" envDefault:""`
 	JWTAuthUserProperty       string   `env:"FLAGR_JWT_AUTH_USER_PROPERTY" envDefault:"flagr_user"`
+	JWTAuthSigningMethod      string   `env:"FLAGR_JWT_AUTH_SIGNING_METHOD" envDefault:"HS256"`
```

Comment:
Let's add a comment for possible values of signing methods.

The whole file will be served as the doc at https://checkr.github.io/flagr/#/flagr_env

Reply:
Also, can you help change the comments above of the pattern of how to use JWT auth?

Reply:
Sure! I missed this comment 👍